test/controllers/msb/SendTheLargestAmountsOfMoneyControllerSpec.scala

```diff
@@ -89,47 +89,15 @@ class SendTheLargestAmountsOfMoneyControllerSpec extends AmlsSpec with MockitoSu
       }
     }
 
-
-    "redirect to Page not found" when {
```

Comment:
Is it worth keeping these tests with the different services, but changing their specification to refer 'SendTheLargestAmountOfMoney...'. This could make it more clear that we are going to this page no matter what.